daxa/process/erosita/_common.py

```diff
@@ -188,7 +269,7 @@ def wrapper(*args, **kwargs):
 
         # I do not love this solution, but this will be what any python errors that are thrown during execute_cmd
         #  are stored in. In theory, because execute_cmd is so simple, there shouldn't be Python errors thrown.
-        #  SAS errors will be stored in process_parsed_stderrs
+        #  eSASS errors will be stored in process_parsed_stderrs
```

Comment:
This is currently not true, because we cannot parse eSASS errors yet, so nothing is being stored in process_parsed_stderrs